### ISSUE TITLE
Instalation problem

### ISSUE THREAD

**Issue by Aleksst95** [paquettg/php-html-parser#235]
Can't install with guzzlehttp/guzzle ^7.0.1

Composer output:
Problem 1
    - Installation request for guzzlehttp/guzzle (locked at 7.0.1, required as ^7.0.1) -> satisfiable by guzzlehttp/guzzle[7.0.1].
    - php-http/guzzle6-adapter 2.x-dev requires guzzlehttp/guzzle ^6.0 -> satisfiable by guzzlehttp/guzzle[6.5.x-dev].
    - php-http/guzzle6-adapter v2.0.0 requires guzzlehttp/guzzle ^6.0 -> satisfiable by guzzlehttp/guzzle[6.5.x-dev].
    - php-http/guzzle6-adapter v2.0.1 requires guzzlehttp/guzzle ^6.0 -> satisfiable by guzzlehttp/guzzle[6.5.x-dev].
    - Conclusion: don't install paquettg/php-html-parser 3.0.1|install guzzlehttp/guzzle 6.5.x-dev
    - Conclusion: don't install guzzlehttp/guzzle 6.5.x-dev
    - paquettg/php-html-parser 3.0.0 requires php-http/guzzle6-adapter ^2.0 -> satisfiable by php-http/guzzle6-adapter[2.x-dev, v2.0.0, v2.0.1].
    - Installation request for paquettg/php-html-parser ^3.0 -> satisfiable by paquettg/php-html-parser[3.0.0, 3.0.1].

**Comment by neokio** [comment]
+1 ... I'm having the same issue.

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: don't install paquettg/php-html-parser 3.0.1
    - Conclusion: remove guzzlehttp/guzzle 7.0.1
    - Installation request for paquettg/php-html-parser ^3.0 -> satisfiable by paquettg/php-html-parser[3.0.0, 3.0.1].
    - Conclusion: don't install guzzlehttp/guzzle 7.0.1
    - paquettg/php-html-parser 3.0.0 requires php-http/guzzle6-adapter ^2.0 -> satisfiable by php-http/guzzle6-adapter[2.x-dev, v2.0.0, v2.0.1].
    - Can only install one of: guzzlehttp/guzzle[6.2.0, 7.0.1].
```

**Comment by deanhouseholder** [comment]
+1

**Comment by mhayta** [comment]
+1 couses problem with laravel 8. please add support

**Comment by hosseinko** [comment]
+1 Please add support

**Comment by paquettg** [comment]
This will be fixed in 3.1.0
